tests/cases/multiple_ubo_update_with_graphics_pipeline.vkscript

```diff
@@ -92,10 +92,12 @@ uniform ubo 1:0 float 0 0.1  0 0 0 \
                         0.8  0 0 0 \
                         0.9  0 0 0 \
                         0.10 0 0 0 \
-                        0.11
+                        0.11 0 0 0
```

Comment:
Extra zeros are just cosmetic, make the test read better, but would have been inserted anyway